src/services/petService.js

```diff
@@ -9,16 +9,20 @@ class PetService {
     return PetModel.findById(petId);
   }
 
-  static async updatePet(petId, threadId) {
-    const pet = await PetService.getPetById(petId);
-    console.log('updatePet in petService: ', pet);
-    pet.threadId = threadId;
-    return await pet.save();
-  }
+  // static async updatePet(petId, threadId) {
```

Comment:
this function should be renamed to updatePetThreadId instead of being commented out, it is used in the creating threads process.

Reply:
Fixed.